awscli/shorthand.py

```diff
@@ -342,7 +359,13 @@ def _second_value(self):
             return self._double_quoted_value()
         else:
             consumed = self._must_consume_regex(self._SECOND_VALUE)
-            return consumed.replace('\\,', ',').rstrip()
+            processed = consumed.replace('\\,', ',').rstrip()
+            return self._should_resolve_paramfiles(processed) if self._resolve_paramfiles else processed
+
+    def _should_resolve_paramfiles(self, val):
```

Comment:
I meant the boolean should be renamed. `_resolve_paramfiles` -> `_should_resolve_paramfiles`

This one should remain `_resolve_paramfile`